03-task-manager/starter/middleware/error-handler.js

```diff
@@ -0,0 +1,12 @@
+const { CustomAPIError } = require("../errors/custom-error");
+
+const errorHandlerMiddleware = (err, req, res, next) => {
+  if (err instanceof CustomAPIError) {
+    return res.status(err.statusCode).json({ msg: err.message });
+  }
+  return res
+    .status(500)
+    .json({ msg: "Something went wrong, please try again" });
```

Comment:
Add the err variable to the JSON so the user can know what condition failed in the validation, for example, when the name is not provided to create the task.
You can do it like this:
return res.status(500).json({ msg: "Something went wrong, try again later", err});

Reply:
Thanks @joseluiz for your kindly feedback. I already changed the code. Please check it. I appreciate it.